src/main/java/io/github/syst3ms/skriptparser/Parser.java

```diff
@@ -24,10 +16,21 @@
 import java.util.jar.JarFile;
 import java.util.jar.Manifest;
 
-public class Main {
+import io.github.syst3ms.skriptparser.log.LogEntry;
+import io.github.syst3ms.skriptparser.log.LogType;
+import io.github.syst3ms.skriptparser.parsing.ScriptLoader;
+import io.github.syst3ms.skriptparser.registration.DefaultRegistration;
+import io.github.syst3ms.skriptparser.registration.SkriptAddon;
+import io.github.syst3ms.skriptparser.registration.SkriptRegistration;
+import io.github.syst3ms.skriptparser.util.ConsoleColors;
+import io.github.syst3ms.skriptparser.util.FileUtils;
+
+public class Parser {
```

Comment:
The renaming to Parser doesn't make 100% sense to me. I get that this name is more descriptive and describes its purpose better, but:
1. If people know that this is the parser they probably know that the Main class serves for the parser as well
2. This adds many unnecessary refactoring.
It's not a huge deal, probably won't even matter.

Reply:
If the parser was the only one project, yeah the ``Main.class`` should be enough. But if you add the parser in another project, it doesn't make sense when you are using it.